src/main/java/com/crafter/springboottechiteasy/services/TelevisionService.java

```diff
@@ -0,0 +1,60 @@
+package com.crafter.springboottechiteasy.services;
+
+import com.crafter.springboottechiteasy.exceptions.RecordNotFoundException;
+import com.crafter.springboottechiteasy.exceptions.RequirementsNotMetException;
+import com.crafter.springboottechiteasy.models.Television;
+import com.crafter.springboottechiteasy.repositories.TelevisionRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TelevisionService {
+
+    private final TelevisionRepository televisionRepository;
+
+    public TelevisionService(TelevisionRepository televisionRepository) {
+        this.televisionRepository = televisionRepository;
+    }
+
+    public List<Television> getAllTelevisions() {
+        return televisionRepository.findAll();
+    }
+
+    public Television getOneTelevision(Long id) {
+            Optional<Television> television = televisionRepository.findById(id);
+            if (television.isPresent()){
+                return television.get();
+            } else {
+                throw new IndexOutOfBoundsException();
```

Comment:
Dit is een logische keus, maar technisch gezien hebben we hier niet te maken met een index, maar met een unieke identifier van de database.